### DISCUSSION TITLE
setup dongle ERROR ss10x1347doo

### DISCUSSION THREAD

**Discussion by batboyboy1990** [mikopbx/Core#570] [Q&A]
DEBUG: pjproject: <?>: ss10x1347doo Error reading CA certificated from buffer

**Comment by boffart** [comment]
Describe the order in which the problem is reproduced

**Comment by batboyboy1990** [comment]
> Describe the order in which the problem is reproduced

The above error appears irregularly! And the dongle works normally! my model is E1750C, appears in the asterisk CLI
